src/test/java/com/alipay/oceanbase/hbase/OHTableClientTest.java

```diff
@@ -57,7 +59,9 @@ public void testNew() throws Exception {
         hTable2.setWriteBufferSize(10000000L);
         assertEquals(10000000L, hTable2.getWriteBufferSize());
         assertEquals("test", hTable2.getTableNameString());
+        //        assertEquals("n1:test", hTable2.getTableNameString());
         assertEquals("test", new String(hTable2.getTableName()));
+        //        assertEquals("n1:test", new String(hTable2.getTableName()));
```

Comment:
need confirm this comments

Reply:
Already removed. Add a new test case file for multi-namespace testing.